lib/cc/cli/engines/install.rb

```diff
@@ -3,6 +3,8 @@
 module CC
   module CLI
     module Engines
+      include CC::Analyzer
```

Comment:
This `include` should probably be nested under `Install` so that this doesn't override the defaults